packages/modular-scripts/src/cli.ts

```diff
@@ -59,9 +62,7 @@ function isYarnInstalled(): boolean {
 
 type PackageType = 'app' | 'view' | 'root'; // | 'package', the default
 
-export interface PackageJson {
-  name: string;
-  private?: boolean;
+export type ModularPackageJson = PackageJson & {
```

Comment:
I can't recollect why this was previously exported, and I can't find an import for this type elsewhere anywhere now. Remove this export?

Reply:
👍